### ISSUE TITLE
Accessibility Issue: Violation of WCAG 2 AA contrast ratio thresholds on page /exercises-oss-jekyll

### ISSUE THREAD

**Issue by Moro-Code** [sara-sabr/ITStrategy#714]
**Description**
Elements have insufficient colour contrast of 3.81 (foreground color: #e83e8c, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

**Element(s) Information**  
*Page*:  https://sara-sabr.github.io/ITStrategy/exercises-oss-jekyll.html
*CSS Selector(s) (if possible and applicable)*:  
```p:nth-child(21) > .language-plaintext:nth-child(2)```
```li:nth-child(1) > .language-plaintext```
```p:nth-child(38) > .language-plaintext```
```p:nth-child(42) > .language-plaintext```
```p:nth-child(50) > .language-plaintext```
```p:nth-child(56) > .language-plaintext```
```p:nth-child(65) > .language-plaintext```
```p:nth-child(74) > .language-plaintext:nth-child(1)```
```p:nth-child(74) > .language-plaintext:nth-child(2)```
```p:nth-child(77) > .language-plaintext```
```p:nth-child(84) > .language-plaintext```
```p:nth-child(87) > .language-plaintext```
```.coffeescript.hljs > .hljs-regexp:nth-child(6)```

*Additional Identifier(s) (if possible and applicable)*:  ```<code class="language-plaintext highlighter-rouge">$PWD</code>```
*Additional Details*:  

**Screenshot(s) (if possible)**

This is an example 

<img width="1035" alt="Screen Shot 2020-01-30 at 2 21 17 PM" src="https://user-images.githubusercontent.com/26819992/73482462-cd1ccd80-436b-11ea-9e53-877daacca818.png">

**Additional information or thoughts**
As a result of scan #675
